### PR TITLE
Update VFS/VSP a11y processes

### PR DESCRIPTION
**Pull request by 1Copenut**
* Adding information about the testing accessibility specialists will do
* Moved screen reader testing to recommended testing, with a note that teams should do these tests if possible

### CODE REVIEW

File: platform/working-with-vsp/roles-and-responsibilities/accessibility.md [hunk 1]
```diff
@@ -1,29 +1,87 @@
...
+  - Provide documentation in [TestRail](https://dsvavsp.testrail.io/index.php?/suites/view/14&group_by=cases:section_id&group_order=asc) or [Github](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/accessibility-test-cases-template.md) certifying tests were run
+  - Fix accessibility issues in application code and large changes to static content, including content generated by Drupal or Markdown.
+- Reduce the backlog of post-launch accessibility improvements over time, as prioritized by their DEPO product lead
+
+## Collaboration cycle for VFS teams with an accessibility specialist
+
+These are the expected responsibilities and division of labor for teams with one or more accessibility specialists. This section will outline each collaboration cycle meeting.
+
+- **[Project kickoff](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle.md#project-kickoff)**
```
Comment (shiragoodman) [inline platform/working-with-vsp/roles-and-responsibilities/accessibility.md:40]: Capitalize Kickoff

File: platform/working-with-vsp/roles-and-responsibilities/accessibility.md [hunk 2]
```diff
@@ -1,29 +1,87 @@
...
+
+  - VFS accessibility specialists are encouraged to attend the project kickoff meeting. Project kickoffs are a good opportunity for teams to ask questions and learn about the new product or feature.
+  - DEPO platform will collect specialist feedback (if any) and provide it to the VFS product team
+
+- **[Design intent](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle.md#design-intent-collaboration)**
```
Comment (shiragoodman) [inline platform/working-with-vsp/roles-and-responsibilities/accessibility.md:45]: Design Intent Collaboration*

File: platform/working-with-vsp/roles-and-responsibilities/accessibility.md [hunk 3]
```diff
@@ -1,29 +1,87 @@
...
+
+  - VFS accessibility specialists are encouraged to attend the design intent meeting
+  - DEPO platform will collect specialist feedback and provide it to the VFS product team. DEPO platform will comment or add guidance as needed.
+  - VFS specialists will answer questions and consult on accessibility issues
+  - VFS specialists will work with DEPO platform if there is a conflict or further collaboration is needed
+
+- **[Usability testing prep](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle.md#usability-testing-prep)**
```
Comment (shiragoodman) [inline platform/working-with-vsp/roles-and-responsibilities/accessibility.md:52]: Usability Testing Prep*

File: platform/working-with-vsp/roles-and-responsibilities/accessibility.md [hunk 4]
```diff
@@ -1,29 +1,87 @@
...
+- **[Staging review](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle.md#staging-review)**
+
+  - VFS accessibility specialists are encouraged to attend the staging review meeting
+  - VFS specialists will conduct the staging review using DEPO platform's [staging review guidance](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md) after product teams have completed their foundational accessibility testing
+  - VFS will enter issue tickets into a ZenHub epic
+  - DEPO platform will review the feedback tickets and discuss any questions with the VFS product team
+  - DEPO platform will deliver findings and notify the product team of any launch blocking issues
+  - VFS specialists will answer questions and help the product team resolve issues
+
+- **[Full accessibility and 508 office audit](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle.md#full-accessibility-and-508-office-audit)**
```
Comment (shiragoodman) [inline platform/working-with-vsp/roles-and-responsibilities/accessibility.md:67]: Full Accessibility and 508 Office Audit*

File: platform/accessibility/guidance/post-launch-audit-processes.md [hunk 5]
```diff
@@ -14,7 +14,10 @@ The post-launch audit will focus on manual testing with mobile and assistive tec
...
-* Issues will be triaged according to their severity. Some issues will be launch blockers. Others can be fixed post-launch. Teams should review the [accessibility defect severity rubric](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md) if they have questions about issue severity or impact.
-* The accessibility specialist will communciate the application or content's readiness for review to the VA 508 office. The specialist will list known issues and serve as a liaison between the requesting team and the VA 508 representative.
-* Accessibility specialists will work with stakeholders as issues are ready to be re-tested. When an issue has been resolved, the specialist or project manager should close it with a brief comment like _"Issue has been tested, and success criteria met. Moving to close."_
+* Issues will be collected as issues in a new ZenHub epic. The accessibility specialist will assign this epic to the project manager (PM) after the audit is complete.
```
Comment (andreahewitt-odd) [inline platform/accessibility/guidance/post-launch-audit-processes.md:17]: I think you mean product manager here

Comment (andreahewitt-odd) [reply platform/accessibility/guidance/post-launch-audit-processes.md:17]: Actually you mention Project Manager many places shere I think you mean product

File: platform/accessibility/guidance/staging-review-processes.md [hunk 6]
```diff
@@ -45,15 +45,22 @@ OR
...
 
-After you request a staging review, the accessibility specialist reviews your work and documents any defects in GitHub/ZenHub tickets. Accessibility issues should be fixed according to the following schedule:
+After you request a staging review, the accessibility specialist will review your foundational tests. When the foundational tests have been verified, the specialist will review your product using several browser and screen reader combinations:
+
+- Windows 10 + IE11/Chrome + JAWS
+- Windows 10 + Firefox + NVDA
+- MacOS + Safari + VoiceOver
+- iOS + Safari + Mobile VoiceOver (as time permits)
+
+The accessibility specialist attaches the tickets to a single ZenHub epic and assigns the epic to the person who requested the staging review. During the staging review meeting, you and your team can ask questions about any of the issues identified. Applications are not considered Section 508 compliant until all severity 0, 1, 2, and 3 issues are remediated. Severity 4 items include minor issues that must be fixed, enhancements, and research.
```
Comment (andreahewitt-odd) [inline platform/accessibility/guidance/staging-review-processes.md:55]: Are applications 508 compliant if they have severity 4 items? I was under the impression that they aren't but this reads like they are.

File: platform/accessibility/guidance/staging-review-processes.md [hunk 7]
```diff
@@ -13,16 +13,16 @@ Staging reviews test the [happy path](https://en.wikipedia.org/wiki/Happy_path)
 - [ ] Color tests for proper contrast and colorblindness - [read more about contrast and colorblindness](#color-tests)
 - [ ] Content zoomed to 200%, 300%, 400% - [read more about zoomed layouts](#content-resize-check)
 - [ ] Keyboard navigation - [read more about navigating with the keyboard](#keyboard-navigation-check)
-- [ ] Screen reader tests (VoiceOver + Safari or NVDA + Firefox) - [read more about getting started with screen readers](#screen-reader-tests)
 
 ## Recommended accessibility tests
```
Comment (emilywaggoner) [inline platform/accessibility/guidance/staging-review-processes.md:17]: Rachael mentioned calling these "Advanced accessibility tests" and the previous section "Baseline" - Just putting that there for you to consider, not a required change I don't think.

File: platform/working-with-vsp/roles-and-responsibilities/accessibility.md [hunk 8]
```diff
@@ -1,29 +1,87 @@
...
+- Reduce the backlog of post-launch accessibility improvements over time, as prioritized by their DEPO product lead
+
+## Collaboration cycle for VFS teams with an accessibility specialist
+
+These are the expected responsibilities and division of labor for teams with one or more accessibility specialists. This section will outline each collaboration cycle meeting.
+
+- **[Project Kickoff](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle.md#project-kickoff)**
+
+  - VFS accessibility specialists are encouraged to attend the project kickoff meeting. Project kickoffs are a good opportunity for teams to ask questions and learn about the new product or feature.
+  - DEPO platform will collect specialist feedback (if any) and provide it to the VFS product team
```
Comment (emilywaggoner) [inline platform/working-with-vsp/roles-and-responsibilities/accessibility.md:43]: What do you think about saying "...collect and review specialist feedback..."?

File: platform/working-with-vsp/roles-and-responsibilities/accessibility.md [hunk 9]
```diff
@@ -1,29 +1,87 @@
...
+
+- **[Design Intent](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle.md#design-intent-collaboration)**
+
+  - VFS accessibility specialists are encouraged to attend the design intent meeting
+  - DEPO platform will collect specialist feedback and provide it to the VFS product team. DEPO platform will comment or add guidance as needed.
```
Comment (emilywaggoner) [inline platform/working-with-vsp/roles-and-responsibilities/accessibility.md:48]: Same thought as above - "...collect and review specialist feedback..."

**Review by emilywaggoner — APPROVED** [review]
Left a couple of minor notes throughout, otherwise I think it looks good. Thanks!

File: platform/accessibility/guidance/staging-review-processes.md (hunk 6 above)
Comment (1Copenut) [reply platform/accessibility/guidance/staging-review-processes.md:55]: Some of the 508 severity 4 items in the past were recommendations that didn't mean the app was out of compliance. I've added a `508-enhancement` tag to capture these things that don't rise to the level of defects.

File: platform/accessibility/guidance/post-launch-audit-processes.md (hunk 5 above)
Comment (1Copenut) [reply platform/accessibility/guidance/post-launch-audit-processes.md]: Did a full text find and replace, got all of the instances changed to product managers.

File: platform/working-with-vsp/roles-and-responsibilities/accessibility.md (hunk 8 above)
Comment (1Copenut) [reply platform/working-with-vsp/roles-and-responsibilities/accessibility.md]: 👍

File: platform/working-with-vsp/roles-and-responsibilities/accessibility.md (hunk 9 above)
Comment (1Copenut) [reply platform/working-with-vsp/roles-and-responsibilities/accessibility.md]: 👍

File: platform/accessibility/guidance/staging-review-processes.md (hunk 7 above)
Comment (1Copenut) [reply platform/accessibility/guidance/staging-review-processes.md:17]: I like that. I fudged a little bit and used the term "foundational", but I've been baking that into conversations so feel it works here too.

Comment (1Copenut) [reply platform/accessibility/guidance/staging-review-processes.md]: 👍

File: platform/accessibility/guidance/post-launch-audit-processes.md [hunk 10]
```diff
@@ -13,8 +13,58 @@ The post-launch audit will focus on manual testing with mobile and assistive tec
...
+- [ ] Content zoomed to 200%, 300%, 400% - [read more about zoomed layouts](#content-resize-check)
+- [ ] Keyboard navigation - [read more about navigating with the keyboard](#keyboard-navigation-check)
+
+### Advanced accessibility tests (recommended)
+
+- [ ] Screen reader tests (VoiceOver + Safari or NVDA + Firefox) - [read more about getting started with screen readers](#screen-reader-tests). Teams are encouraged to conduct their own screen reader tests as part of the software development lifecycle (SDC). _DEPO Platform acknowledges this testing is aspirational._
+- [ ] Axe-coconut scans in daily development - [read more about the axe-coconut experimental plugin](https://www.deque.com/blog/test-leading-edge-accessibility-axe-coconut-axe-core-3-0/)
+- [ ] WAVE tool spot checks - [read more about the WebAIM WAVE tool](https://wave.webaim.org/)
+
+## What to expect from accessibility specialists during post-launch audit audit
```
Comment (rroueche) [inline platform/accessibility/guidance/post-launch-audit-processes.md:33]: Just a typo - you have "audit" written twice

**Pull request merged**